ew/static/status.py

```diff
@@ -88,8 +88,8 @@
     EwStatusEffectDef(
         id_status=ewcfg.status_thinned_id,
         time_expire=ewcfg.time_expire_high,
-        str_describe="They are gassed up on paint thinner.",
-        str_describe_self="You are gassed up on paint thinner."
+        str_describe="They are gassed up on paint fumes.",
+        str_describe_self="You are gassed up on paint fumes."
```

Comment:
I'm gonna go brain mode and, as a suggestion, say this should be "organic vapors". Makes more sense, because paint thinner doesn't really release "paint fumes" per-se.